src/component/Header/Header.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {FC, useState} from 'react';
+import './header.scss';
+import './HeaderResponsive.scss';
+import {Logo} from "./Logo";
+import {UserProfile} from "./UserProfile";
+import { NavLink } from 'react-router-dom';
+export const Header: FC = () => {
+    // Start click function for menu
+    const [menuToggled, setMenuToggled] = useState(false);
+    const ToggleMenu = () => {menuToggled ? setMenuToggled(false) : setMenuToggled(true);}
+    // End click function for menu
+    // Start dropdown function for menu
+    const [dropdownToggled, setDropdownToggled] = useState(false);
+    const DropdownToggle = () => {dropdownToggled ? setDropdownToggled(false) : setDropdownToggled(true);}
+    // End dropdown function for menu
+    return (
+        <div id="header" className="w-100 main-bg">
+            <div className="container">
+                <div className="header-content-wrap">
+                    {/*End Component*/}
+                    <Logo/>
+                    {/*End Logo Component*/}
+                    {/*Start Header Navigation*/}
+                    <div className={`header-navigation ${menuToggled ? "active" : ""}`}>

```

Comment:
use nav tag and you can use header-navigation class on that tag with id of nav like nav#nav

Reply:
added a nav tag instead of div with nav ID